### ISSUE TITLE
[BUG] Refine development server long loading time

### ISSUE THREAD

**Issue by jatwong** [refinedev/refine#3630]
### Describe the bug

I was following the refine quick start tutorial to get a taste of using refine. 
My files are within in the Linux system (Debian/WSL2).

CRA takes about 2 seconds to load, while Refine app takes about 10 seconds to load.

### Steps To Reproduce

1. npm run dev on a completely fresh install of refine-create app

### Expected behavior

Web dev server load is longer than CRA dev server load

### Desktop

-Windows 10
-Chrome
-WSL2/Debian v11
-VS Code

**Comment by aliemir** [comment]
Hey @jatwong, thank you for opening the issue 🙏 This is obviously not an intended behavior 😅 We'll investigate the issue and test it out in various environments. Thanks for sharing an info about yours, can you please also provide the response of `refine whoami` command? 🙏

**Comment by jatwong** [comment]
Hey @aliemir ,
I posted this for @alicanerdurmaz on the Discord already, but I'll post it here too.

![image](https://user-images.githubusercontent.com/111151026/219979887-644856f9-fdbe-4ea9-87c9-8133490cc04a.png)

**Comment by alicanerdurmaz** [comment]
@jatwong Hi again, can you set `UPDATE_NOTIFIER_IS_DISABLED` env variable to and can you tell me the results?

You can set it like this.
```
// package.json
"dev": "UPDATE_NOTIFIER_IS_DISABLED=true refine start",
```

```
// .env
UPDATE_NOTIFIER_IS_DISABLED=true
```

**Comment by jatwong** [comment]
I've added it to package.json, the load time is about 8 sec, instead of 10.

**Comment by alicanerdurmaz** [comment]
> I've added it to package.json, the load time is about 8 sec, instead of 10.

Does it take 8 seconds on all runs?

**Comment by jatwong** [comment]
It seems to fluctuate between 8-10 seconds

**Comment by alicanerdurmaz** [comment]
Thank you @jatwong, we are investigating the issue.

**Comment by hongfanmeng** [comment]
I have a long loading time in refine-nextjs also, take about 10s.
Platform: WSL2 (Ubuntu 22.04)

**Comment by aliemir** [comment]
This issue might got fixed after the #4355 PR which released at `@refinedev/mui@4.16.4`.
